src/main/java/com/iota/iri/network/replicator/Replicator.java

```diff
@@ -7,16 +7,9 @@
 
 
 /**
- * This class manages a set of Source and Sink pool workers {@link ReplicatorSourceProcessor}
- * and {@link ReplicatorSinkProcessor}. The workers are managed by the Pool manager threads
- * {@link ReplicatorSourcePool} and {@link ReplicatorSinkPool}
- * <br>
- * A **Sink** is basically a received end point for TCP where all  the peers send
```

Comment:
I think these are mixed up - see the `ReplicatorSinkProcessor: out.write()`  & `ReplicatorSourceProcessor: stream.read()`